pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -82,7 +82,7 @@
 , withSvtav1 ? withFullDeps && !stdenv.isAarch64 # AV1 encoder/decoder (focused on speed and correctness)
 , withTensorflow ? false # Tensorflow dnn backend support
 , withTheora ? withHeadlessDeps # Theora encoder
-, withV4l2 ? withFullDeps && !stdenv.isDarwin # Video 4 Linux support
+, withV4l2 ? withSmallDeps && !stdenv.isDarwin # Video 4 Linux support
```

Doe V4L require a headed system to be useful? If not:

```suggestion
, withV4l2 ? withHeadlessDeps && !stdenv.isDarwin # Video 4 Linux support
```

---

I don't know.. you typically use a webcam on a desktop or laptop.

---

I found myself reaching for `v4l2` on a headless Pi the other day so that I could stream video from a webcam over a network.

---

Done.